### PR TITLE
Make <img> in 1-Index-html translatable

### PR DESCRIPTION
**Pull request by ignotus666**
**Short description of changes**
This adds an option to the po4a script to make the content of the `` tag translatable - it was being skipped by po4a and made it impossible to show localised screenshots of the main mixer window on the homepage. It also makes it possible to easily add an arbitrary number of options in future if needed.

**Context: Fixes an issue? Related issues**
This problem was pointed out by @henkdegroot on Discord.

**Status of this Pull Request**
Tested and working

**What is missing until this pull request can be merged?**
Any Weblate PRs have to be merged first.

**Does this need translation?**

NO.

## Checklist

- [ ] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [ ] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [ ] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

File: wiki/en/misc/1-index.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Jamulus lets you play, rehearse, or jam with your friends, your band, or anyone
   <div class="fx-col-100-xs">
     <figure class="mainbannerfig">
       <a href="wiki/Getting-Started">
-      <img src="{% include img/en-screenshots/main-screen-medium.inc %}" style="border: 5px solid grey;" id="jamulusbanner" loading="lazy" alt="A screenshot of the main mixer window showing five people from different countries connected."/>
+      <img src="{% include img/en-screenshots/main-screen-medium.inc %}" style="border: 5px solid grey;" id="jamulusbanner" loading="lazy" alt="A screenshot of the main mixer window showing several people from different countries connected."/>
```
Comment (ann0see) [inline wiki/en/misc/1-index.md]: Should probably not be part of this PR though.

File: _po4a-tools/po4a-create-all-targets.sh [hunk 2]
```diff
@@ -99,16 +99,17 @@ process_with_po4a () {
...
+        # Determine file format to be used and set options (configured as an array to allow adding an arbitrary number of them)
+        OPTION=()
         if [ $ext == yml ] ; then
             FILE_FORMAT=yaml
-            OPTION="skip_array"
+            OPTION=("skip_array")
         elif [[ $ext == html || "$filename" == *'-index' ]] ; then # '-index.md' has a markdown extension but is actually html and should be processed as such by po4a
             FILE_FORMAT=xml
-            OPTION="ontagerror=warn"
+            OPTION=("ontagerror=silent" "attributes=<img>src <img>alt")
```
Comment (ann0see) [inline _po4a-tools/po4a-create-all-targets.sh]: ontagerror=silent sounds like making debugging harder.

**Review by ann0see — CHANGES_REQUESTED** [review]

File: wiki/en/misc/1-index.md (hunk 1 above)
Comment (ignotus666) [reply wiki/en/misc/1-index.md]: Ok, I'll make a separate PR.

File: _po4a-tools/po4a-create-all-targets.sh (hunk 2 above)
Comment (ignotus666) [reply _po4a-tools/po4a-create-all-targets.sh]: Ok - I just remembered @softins made a change that got rid of the errors, so I'll leave it with `ontagerror=warn` then.

File: _po4a-tools/po4a-create-all-targets.sh [hunk 3]
```diff
@@ -120,7 +121,7 @@ process_with_po4a () {
             --localized "$targ_doc" \
             --localized-charset "UTF-8" \
             --no-deprecation \
-            --option "$OPTION" \
+            "${OPTION[@]/#/--option=}" \
```
Comment (pljones) [inline _po4a-tools/po4a-create-all-targets.sh:124]: W00tz array magic. 😄

**Review by pljones — APPROVED** [review]

**Review by ann0see — COMMENTED** [review]
It seems hacky to me, but well...

**Review by ann0see — APPROVED** [review]

**Comment by pljones** [comment]
Merging tools (or anything not translatable on Weblate is fine without locking Weblate).

**Comment by ignotus666** [comment]
Yeah, originally this PR contained a change to index.md that was later removed.
